pkg/storage/storage.go

```diff
@@ -114,6 +114,19 @@ func (s *storageModule) BrokenDevices() []pkg.BrokenDevice {
 	return s.brokenDevices
 }
 
+func (s *storageModule) Dump() {
+	log.Debug().Int("volumes", len(s.volumes)).Msg("dumping volumes")
+
+	for i := range s.volumes {
+		devices := s.volumes[i].Devices()
+		for id := range devices {
+			d := devices[id]
+			log.Debug().Str("path", d.Path).Str("labem", d.Label).Str("type", string(d.DiskType)).Msg("dump")
```

Comment:
```suggestion
			log.Debug().Str("path", d.Path).Str("label", d.Label).Str("type", string(d.DiskType)).Send()
```

Reply:
.Send() allow to print the log without the need for a message body. So it just print a line with the arguments.